xds/src/test/java/io/grpc/xds/internal/sds/trust/SdsX509TrustManagerTest.java

```diff
@@ -84,180 +83,6 @@ public void emptySanListContextTest() throws CertificateException, IOException {
     trustManager.verifySubjectAltNameInChain(certs);
   }
 
-  @Test
```

Comment:
Looking at the latest findings in https://github.com/grpc/grpc-java/issues/7166#issuecomment-652536045 where we will use match_subject_alt_names with EXACT_MATCH most of these tests will be applicable (after making the change). Would it be better to `@Ignore` them instead of just deleting them?

Reply:
I used `@Ignore` for some other tests. These tests here can not compile because `addVerifySubjectAltName()` does not exist for v3 api.

Reply:
Sounds good. Can you add a TODO comment (assigned to me) to remind that these 11(?) tests need to be brought back when the new `match_subject_alt_names` field is implemented?

Reply:
I added a TOOD referring the issue here https://github.com/grpc/grpc-java/pull/7169/files/6f32fb0eec0589cd4d606418de35ca2a85cfb855#diff-9efb11763826a58994b5e41a318c58d8R54 which has discussions on `match_subject_alt_names`

Reply:
Okay I added an item in the issue so I can track it there.